src/app.ts

```diff
@@ -213,3 +214,5 @@ if (ENV.DEBUG_THEME) {
   }
   themeFunc().catch(err => console.error(err))
 }
+
+initDeviceSettings().catch(err => console.error(err))
```

Comment:
Please avoid `console.error` in general. This is the red box of doom, which should only be used for genuine emergencies. Just use console.log or console.warn or something.